it/src/test/java/com/linecorp/centraldogma/it/updater/CentralDogmaBeanTest.java

```diff
@@ -113,13 +120,19 @@ public void overrideSettings() throws Exception {
         await().atMost(5, TimeUnit.SECONDS).until(() -> property.getFoo() == 200);
         assertThat(property.getBar()).isEqualTo("YY");
         assertThat(property.getQux()).containsExactly("100", "200");
+        assertThat(property.getRevision()).isEqualTo(new Revision("2"));
     }
 
     @CentralDogmaBean(project = "a", repository = "b", path = "/c.json")
     static class TestProperty {
         int foo = 10;
         String bar = "20";
         List<String> qux = ImmutableList.of("x", "y", "z");
+        Revision revision;
+
+        public Revision getRevision() {
+            return revision;
```

Comment:
The field `revision` isn't used at all actually. We could just return null here and remove the field `revision`.